settings/settings.php

```diff
@@ -84,7 +86,7 @@ function is_plugin_enabled( $plugin ) {
  */
 function render_graphql_endpoint() {
 	// Match logic in \WPGraphQL\Router:
-	// https://github.com/wp-graphql/wp-graphql/blob/df2c5b2556fcbd69983d96d4e906572d835c0832/src/Router.php
+	// https://github.com/wp-graphql/wp-graphql/blob/df2c5b2556fcbd69983d96d4e906572d835c0832/src/Router.php.
```

Comment:
this can be a little disruptive, what is forcing this?

Reply:
This is coming from the default OOB coding standards. We could turn this off, but having had this in the Block Data API and Block Governance plugins it's become a force of habit now. It is disruptive at first for sure.

Reply:
I only mean specifically for this link. When I click on it, it 404s because of the trailing .

Reply:
Ah, I see. I'll leave this rule as is as it keeps it consistent with the other plugin, but if it gets disruptive we can revert it.